go/service/main.go

```diff
@@ -184,6 +187,17 @@ func (d *Service) writeServiceInfo() error {
 	return rtInfo.WriteFile(path.Join(runtimeDir, "keybased.info"))
 }
 
+func (d *Service) checkTracksEveryHour() {
```

Comment:
I've suggested rate-limiting too, but FWIW, on the Electron side I plan to send one RPC per minute when *any* tracker window is open, not one RPC per window per minute.